src/js/containers/bulkDownload/accounts/AccountDataContainer.jsx

```diff
@@ -74,6 +77,42 @@ export class AccountDataContainer extends React.Component {
             });
     }
 
+    setFederalList(agencyCode) {
+        this.setState({
+            inFlight: true
```

Comment:
At this point, we're not disabling the filters or displaying a loading message while the requests are in flight, so you can actually remove `inFlight` from the state, here,  and from `setAgencyList`.